cmd/flags/passphrase.go

```diff
@@ -102,7 +103,15 @@ func ReadPassphraseInputWithOpts(withConfirmation bool) (string, error) {
 	return passphrase, nil
 }
 
+func runningInMsys() bool {
+	ms := os.Getenv("MSYSTEM")
+	return ms != ""
```

Comment:
Beware, you are not checking for errors here. Also, ou may want to consider `os.LookupEnv()` instead. 

https://pkg.go.dev/os#LookupEnv

Reply:
Sorry, my mistake, it's a false statement. Ignore this.